docs/api/_index.md

```diff
@@ -698,6 +698,9 @@ _This experimental endpoint is disabled by default and can be enabled via the `-
 
 _Requires [authentication](#authentication)._
 
+> **Note:** When using `curl` send the request body from a file, ensure that you use the `--data-binary` flag instead of `-d`, `--data`, or `--data-ascii`.
+> The latter options do not preserve carriage returns and newlines.
```

Comment:
Reference for this claim? I cannot see that in the man page.

Reply:
Ah, I misread. It's actually documented: `Posting data from a file named from a file like that, carriage returns and newlines will be stripped out.`

Reply:
Yeah I honestly read the page three times and missed it each time haha.